storage/innobase/xtrabackup/src/xtrabackup.cc

```diff
@@ -1532,6 +1531,15 @@ check_if_param_set(const char *param)
 	return param_set.find(param) != param_set.end();
 }
 
+/* distory the argument */
```

Comment:
Please improve this. SOmething like

Mask the argument value. This is to avoid showing secret data on command line output